README.md

```
@@ -43,74 +43,98 @@ Installation
 
 If you downloaded the source code from GitHub, read our guide on [installing PrestaShop for development][install-guide-dev]. If you intend to install a production shop, make sure to download the latest version from [our releases page][download], then read the [install guide for users][install-guide].
 
-Docker compose
---------
+## 🐳 Docker Development Environment
```

markdownlint-cli2 (0.17.2)

Heading style
Expected: setext; Actual: atx

(MD003, heading-style)